src/components/Wizard/WizardHeader.js

```diff
@@ -1,40 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
+import { noop, Modal, Icon } from '../../index';
 
 /**
  * WizardHeader component for Patternfly React
  */
-const WizardHeader = ({ children, className, embedded, title, ...rest }) => {
-  const classes = classNames({ 'wizard-pf-header': !embedded }, className);
-
-  if (embedded) {
-    return (
-      <h2 className={classes} {...rest}>
-        {title}
-      </h2>
-    );
-  }
-  return (
-    <div className={classes} {...rest}>
-      <h4 className="wizard-pf-title">{title}</h4>
-    </div>
-  );
-};
+const WizardHeader = ({ onClose, title, ...props }) => (
+  <Modal.Header {...props}>
+    <button
+      className="close"
+      onClick={onClose}
```

Comment:
since close is "optional", do we want to hide this if it is not present?  Seems odd to show a close icon if close doesn't work.

Reply:
I agree, maybe we should render the "close" button only if `onClose` is defined.

Reply:
Should probably set onClose to required in propTypes and remove the default noop. to enforce this.  My thinking was that there should always be a way to close it as well.